docs/content/dagster-cloud/deployment/serverless.mdx

```diff
@@ -94,41 +94,86 @@ Any dependencies specified in either `requirements.txt` or `setup.py` will be in
 
 ---
 
-## Customizing the container
+## Customizing the runtime environment
 
-Many apps will work fine with the default Dagster Cloud Serverless setup. However, some apps may need to make changes to the base image, either to use a different OS, different Python version, or install some native dependencies. You can customize your image through our Serverless lifecycle hooks, and/or changing the base image.
+Dagster Cloud Serverless packages your code as [PEX](https://pex.readthedocs.io) files and deploys them on Docker images. Many apps will work fine with the default Dagster Cloud Serverless setup. However, some apps may need to make changes to the runtime environment, either to include data files, use a different OS, different Python version, or install some native dependencies. You can customize the runtime environment using various methods described below.
 
-### Lifecycle hooks
+### Including data files
 
-This method is the easiest to set up, and does not require setting up any additional infrastructure.
+To add data files to your deployment, use the [Data Files Support](https://setuptools.pypa.io/en/latest/userguide/datafiles.html) built into Python's setup.py. This requires adding a `package_data` or `include_package_data` keyword in the call to `setup()` in `setup.py`.
```

Comment:
linking to or showing an example here would be pretty helpful i think

Reply:
+1